app/views/defence_requests/form_partials/_solicitor_details.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="form-group">
   <%= f.label :solicitor_name, t('full_name') %>
-  <%= f.text_field :solicitor_name, class: 'text-field text-field-wide' %>
+  <%= f.text_field :solicitor_name, disabled: !@policy.edit_solicitor_details?, class: 'text-field text-field-wide' %>
```

Comment:
We're using pundit policies everywhere to handle view logic. I don't think this will bring us much other than extra complexity.